core/lib/config/src/configs/experimental.rs

```diff
@@ -106,4 +106,8 @@ pub struct ExperimentalVmConfig {
     /// the new VM doesn't produce call traces and can diverge from the old VM!
     #[serde(default)]
     pub state_keeper_fast_vm_mode: FastVmMode,
+
+    /// Fast VM mode to use for gas estimation in the API server.
+    #[serde(default)]
+    pub api_fast_vm_mode_for_gas_estimation: FastVmMode,
```

Comment:
I haven't added corresponding option for calls and transaction execution; they can be added iteratively. Not sure what's the best approach here.

Reply:
Probably one variable for all 3 cases is enough?

Reply:
Do you think it should be used for all 3 cases from the start? Or should it be used for gas estimation rn, and then extended to other 2 cases? cc @RomanBrodetski

Reply:
IMO it's fine to use it for all 3. If we encounter problems with any of them, we can temporarily disable the new VM mode. But if you have any reasons to keep it only for gas estimation for now, I'm more than fine with keeping it

Reply:
No particular reason; I'll simplify configuration then.